src/models/Revisions.php

```diff
@@ -22,27 +22,27 @@
  */
 class Revisions implements DestroyableInterface
 {
-    /** @var int MIN_DELTA the min number of characters different between two versions to trigger save */
-    private const MIN_DELTA = 100;
-
     private Db $Db;
 
+    private Config $Config;
+
     private AbstractEntity $Entity;
 
     public function __construct(AbstractEntity $entity)
     {
         $this->Entity = $entity;
         $this->Db = Db::getConnection();
+        $this->Config = new Config();
```

Comment:
Config object should be passed in the constructor (dependency injection). Also helps making 1 less sql request.

Reply:
but that's ok for now